### DISCUSSION TITLE
How to remove or change color focus within?

### DISCUSSION THREAD

**Discussion by MariuzM** [mantinedev/mantine#6249] [Q&A]
![image](https://github.com/mantinedev/mantine/assets/36937039/efaa8a2a-8069-4198-98da-8476616d933c)

I have this modal and sometimes when i open it i get this focus automatically, how can i remove and also how can i control its color?

**Comment by rtivital** [comment]
You can remove focus ring by applying focus styles to the close button using Styles API. Note that it will be accessibility violation.

**Reply by MariuzM** [reply]
not sure if this is a bug but its just sometimes when i open modal it has it highlighted i'm sure that's not how it should work?

**Reply by rtivital** [reply]
Focus ring is visible when the user interacts with the website using keyboard.
